### DISCUSSION TITLE
Jellyfin login disclaimer not showing on reverse proxy

### DISCUSSION THREAD

**Discussion by rodude123** [9p4/jellyfin-plugin-sso#148] [Q&A] [answered]
So I followed Jellyfin's guide to reverse proxy, and it works, but the login disclaimer section doesn't
I followed your guide to add the login disclaimer like so:
![image](https://github.com/9p4/jellyfin-plugin-sso/assets/10896262/3d1c0dca-13ca-495f-bfba-c55944ae046f)
it works when I use local IP (192.168.0.130:8096/web/index.html)

![image](https://github.com/9p4/jellyfin-plugin-sso/assets/10896262/617ccd3f-968d-4ba2-a67e-4a317ab4c252)

But I don't see the button when I go to my reverse proxy.

![image](https://github.com/9p4/jellyfin-plugin-sso/assets/10896262/737c82f2-2404-4ed2-8f39-171641db55df)

(P.S. I've hidden my real URL for privacy reasons, but you can assume it's https://jellyfin.example.co.uk/web/index.html.)

Am I doing something wrong?

**Comment by 9p4** [comment, marked as answer]
Does your reverse proxy cache anything?

**Reply by rodude123** [reply]
that should do it I restarted and it worked.
